includes/shortcodes.php

```diff
@@ -1,135 +1,140 @@
+                    $exclude[] = $chair->ID;
+                }
+                if ( !empty( $vice_chair_id ) ) {
+                    $vice_chair = get_post( $vice_chair_id );
+                    $vice_chair = UCF_People_PostType::append_metadata( $vice_chair );
+                    $exclude[] = $vice_chair->ID;
+                }
+                if ( count( $exclude ) > 0 ) {
+                    $args['post__not_in'] = $exclude;
+                }
+            }
+            $people = get_posts( $args );
+            $count = count( $people ) - 1;
+            if ( isset( $chair ) ) :
+        ?>
+                <?php if ( $i % 3 === 0 ) : ?><div class="row"><?php endif; ?>
+                <div class="col-md-4 col-sm-6">
+                    <?php echo BOTUtilities::get_person_markup( $chair, 'Board Chair' ); ?>
+                </div>
+                <?php if ( $i % 3 === 2 ) : ?></div><?php endif; $i++; $count++; ?>
+        <?php
+            endif;
+            if ( isset( $vice_chair ) ) :
+        ?>
+                <?php if ( $i % 3 === 0 ) : ?><div class="row"><?php endif; ?>
+                <div class="col-md-4 col-sm-6">
+                    <?php echo BOTUtilities::get_person_markup( $vice_chair, 'Board Vice Chair' ); ?>
+                </div>
+                <?php if ( $i % 3 === 2 ) : ?></div><?php endif; $i++; $count++; ?>
+        <?php
+            endif;
+            foreach( $people as $person ) :
+                $person = UCF_People_PostType::append_metadata( $person );
+        ?>
+            <?php if ( $i % 3 === 0 ) : ?><div class="row"><?php endif; ?>
+            <div class="col-md-4 col-sm-6">
+                <?php echo BOTUtilities::get_person_markup( $person ); ?>
+            </div>
+            <?php if ( $i % 3 === 2  || $i === $count ) : ?></div><?php endif; $i++; ?>
+        <?php
+            endforeach;
+            return ob_get_clean();
+        }
 
-add_shortcode( 'charter-list', 'botutils_people_group_charter_list_shortcode' );
+        public static function people_group_charter_list_shortcode( $atts, $content="" ) {
+            $none_term = term_exists( 'None', 'people_group' );
+            $terms = get_terms( array(
+                'taxonomy' => 'people_group',
+                'exclude'  => array( $none_term )
+            ) );
+            ob_start();
+        ?>
+            <ul class="list-unstyled document-list">
+            <?php foreach( $terms as $term ) : $charter = get_field( 'people_group_charter', 'people_group_' . $term->term_id ); ?>
+                <?php if( $charter ) : ?>
+                    <li><a class="document" href="<?php echo $charter; ?>"><?php echo $term->name; ?> Committee Charter</a></li>
+                <?php endif; ?>
+            <?php endforeach; ?>
+            </ul>
+        <?php
+            return ob_get_clean();
+        }
+    }
 
+    add_shortcode( 'people-list', 'BOTUtilities::people_list_shortcode' );
```

Comment:
While this probably works, we normally pass in static functions as an array.. I think it's specified in the WordPress style guide. So: `add_shortcode( 'people-list', array( 'BOTUtilities', 'people_list_shortcode' );`